src/transport/SecurePairingSession.cpp

```diff
@@ -107,9 +90,45 @@ CHIP_ERROR SecurePairingSession::Deserialize(SecurePairingSessionSerialized & in
     memset(&serializable, 0, sizeof(serializable));
     deserializedLen =
         Base64Decode(Uint8::to_const_char(input.inner), static_cast<uint16_t>(len), Uint8::to_uchar((uint8_t *) &serializable));
+
     VerifyOrExit(deserializedLen > 0, error = CHIP_ERROR_INVALID_ARGUMENT);
     VerifyOrExit(deserializedLen <= sizeof(serializable), error = CHIP_ERROR_INVALID_ARGUMENT);
 
+    error = FromSerializable(serializable);
+
+exit:
+    return error;
+}
+
+CHIP_ERROR SecurePairingSession::Serializable(SecurePairingSessionSerializable & serializable)
```

Comment:
Yeah, this should be called `Serialize` for symmetry with `Deserialize`

Reply:
This function corresponds to `FromSerializable`. It would output a `serializable` object. Would changing it to `ToSerializable` be clearer?

Reply:
I'd prefer `ToSerializable`, yes.